File: telerik_ui_for_winforms_courseware_chm/Courseware/Treeview/CS/DatabindingTreeView/Databinding/SimpleData.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Telerik.WinControls;
using Telerik.WinControls.UI;

namespace Databinding
{
    public partial class SimpleData : RadForm
    {
        public SimpleData()
        {
            InitializeComponent();
        }

        private void SimpleData_Load(object sender, EventArgs e)
        {
            this.employeesTableAdapter.Fill(this.nwindDataSet.Employees);
            radTreeView1.DataSource = this.nwindDataSet.Employees;
            radTreeView1.DisplayMember = "LastName";
        }
    }
}

File: telerik_ui_for_winforms_courseware_chm/Courseware/Treeview/VB/DatabindingTreeView/Databinding/SimpleData.vb
Public Class SimpleData

    Private Sub SimpleData_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        Me.employeesTableAdapter.Fill(Me.nwindDataSet.Employees)
        radTreeView1.DisplayMember = "LastName"
        radTreeView1.DataSource = Me.nwindDataSet.Employees
    End Sub
End Class
